src/main/webapp/profileScript.js

```diff
@@ -146,3 +147,30 @@ function addRoute(newRoute) {
     document.getElementById("future-routes").appendChild(card);
   }
 }
+
+function showSubmitAvatar() {
+  document.getElementById("submit-avatar").classList.toggle("show");
+}
+
+function showAvatar() {
+  fetch("/profile-image", {
+    method: "GET",
+  })
+    .then((response) => response.json())
+    .then((avatarName) => {
+      try {
+        document.getElementById("avatar-image").innerHTML =
+          '<img src="https://storage.cloud.google.com/user-image-globes/' +
```

Comment:
You should not create HTML as a string. It is better to create an element (using document.createElement), set the properties on it and add it to the child collection. Generating inner HTML like this is very error prone. E.g. if in future avatarname contains a quote or some other special HMTL character this will break.

See https://www.w3schools.com/jsref/dom_obj_image.asp